src/react/ssr.html

```diff
@@ -6038,6 +6038,8 @@ <h2 class="post__comment-author">@username_301 wrote</h2>
 
   </section>
 
+  <script src="react.min.js"></script>
+  <script src="react-dom.min.js"></script>
   <script src="app.js" async></script>
```

Comment:
`async` does not guarantee order unfortunately. You can use `defer` though :-)

`defer` tells browser to download scripts but not to parse them until it has finished parsing the document, then to execute each script in the order seen.